data/sales.txt

```diff
@@ -1284,6 +1284,67 @@ outfitter "sheragi"
 	"Sheragi Anti-Missile"
 	"Small EM Battery"
 	"Small Hybrid Cooling"
+shipyard "successors"
+	"Aaulqra"
+	"Ejniisde"
+	"Kaskhorade"
+	"Kvelqde"
+	"Kvelqde-Khora"
+	"Nnaatej"
+	"Nnesaarau"
+	"Myaede"
+	"Ojde"
+	"Seiitej"
+	"Stolsaqra"
+	"Veusade"
+	"Vujlet"
+outfitter "successors"
+	"Auxiliary Power Reserve"
+	"Primary Power Reserve"
+	"Bimodal Coilgun Turret"
+	`"Ojet" Bimodal Coilgun`
+	`"Sjeja" Kinetic Lance`
+	"High-Velocity Spike"
+	"High-Velocity Spike Rack"
+	"Kinetic Rail Rifle"
+	"Heirloom License"
+	"High Houses License"
+	"Multimodal Armor: Heavy"
+	"Multimodal Armor: Regen"
+	"Multimodal Armor: Resilient"
+	"Multimodal Armor: Keystone"
+	"Overcharged Laser Turret"
+	"Overcharged Pulse Laser"
+	"Pulse Laser Turret"
+	"Drag Cannon"
+	"Successor Cooling"
+	"Successor Fuel Sail"
+	"Successor License"
```

Comment:
Looking through the files, it looks like Omnis usually gives all necessary licenses to the players directly in the Omnis start (start.txt). So the Successor, High Houses, and Heirloom licenses should probably be given to the player directly in that file.